### PR TITLE
[wip] Create English version of all docs

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: translations/en/about.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# jsunderhood
+
+Collective twitter account for developers with new author each week.
+
+This is interesting analog of reddit AMA, but in tweetter with new functions.
```
Comment (kinday) [inline translations/en/about.md]: `tweetter` → `twitter`

File: translations/en/about.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+This account provide the ability to read and discuss controversial and important
```
Comment (kinday) [inline translations/en/about.md]: `provide` → `provides`

File: translations/en/about.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+issues with a weekly changing sponsors.
```
Comment (kinday) [inline translations/en/about.md]: Wha-a-at? You mean “presenters”, aren’t you?

File: translations/en/authoring.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Пожалуйста будьте вежливыми и уважайте ваших читателей. Не следует оскорблять
+или унижать кого-то. Ведите себя профессионально. Уничижительные, сексистские,
+расистские или ксенофобские шутки не привествуются в [@jsunderhood][jsh].
+
+Если вы заметили нарушение, как можно скорее напишите
+письмо [Антону][coc-violations].
+
+[jsh]: https://twitter.com/jsunderhood
+[proposal]: mailto:anton@shuvalov.info?subject=Author’s%20proposal%20for%20@jsunderhood
+[coc-violations]: mailto:anton@shuvalov.info?subject=CoC%20violations%20in%20@jsunderhood
```
Comment (iamstarkov) [inline translations/en/authoring.md:67]: @davydovanton will you translate this file?

**Comment by iamstarkov** [comment]
good job 👍

File: translations/en/authoring.md (hunk 4 above)
Comment (davydovanton) [inline translations/en/authoring.md:67]: didn't have time for this file :disappointed:

Comment (iamstarkov) [inline translations/en/authoring.md:67]: it’s okay

Comment (davydovanton) [inline translations/en/authoring.md:67]: :disappointed:

**Comment by A** [comment]
@iamstarkov This docs need to proofread. But can we merge it to simplify it? It's hard to make pull requests to @davydovanton's PR :D

**Comment by davydovanton** [comment]
I also will be happy to merge this :+1:

File: translations/en/about.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# jsunderhood
+
+Collective twitter account for developers with new author each week.
+
+This is interesting analog of reddit AMA, but in twitter with new functions.
```
Comment (kinday) [inline translations/en/about.md:5]: It is an interesting analog if Reddit’s AMA, but in the Twitter and with new functions.

Comment (kinday) [inline translations/en/about.md]: interesting → an interesting

Comment (kinday) [inline translations/en/about.md:5]: Мне и в русском не нравится слово «функции». Я бы заменил на возможности. А в английском вообще бы написал «with super powers».

File: translations/en/about.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+This account provides the ability to read and discuss controversial and important
```
Comment (kinday) [inline translations/en/about.md:6]: «the» лишний.

File: translations/en/about.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+issues with a weekly.
```
Comment (kinday) [inline translations/en/about.md:7]: «...with a new author every week», да?

**Comment by kinday** [comment]
I’m against merging. The problem is Russian version far from perfection. Gimme some time, I’ll fix Russian one and proofread English one.

**Comment by A** [comment]
@kinday do you have any updates here?)

**Comment by kinday** [comment]
@A, wow, something happened and I forgot about this task. Sorry, will make it later today.

**Comment by kinday** [comment]
The process is not as easy as I wanted it to be. You can track progress and review changes at davydovanton/jsunderhood#2

I expect to edit one more document today and the last one tomorrow.

**Comment by kinday** [comment]
Hey, I’m not able to finish the task in time so I’m stepping aside. Sorry for delaying the whole process.

**Pull request closed without merging**